fintual-oss/paraffin-front
# fix: change routes and fix unauth name

**CHANGES**

- Change localhost:3001 route on Unauth Guard and Auth Context. 
- Add some routes on endpoints utils dictionary.
- Fix Unuth name to Un**a**uth for coherence.  

**IF REPO IS NOT CLONED**
1. Clone Repo
2. Run yarn install
3. Run yarn dev

**QA**
1. Open root ("/").
2. Try to access with button "Ir Camino de Aprendizaje" => Should redirect to Sign In. 
3. Sign In with valid user => Should return to root ("/") with changed Navbar. 
4. Navigate to Curriculums => Should show Learning Units. 
5. Complete and Uncomplete Learning Units => Should change on screen. 
6. Sign Out on Navbar => Should redirect to Sign In Page
7. Try "localhost:3001/curriculum/1:" directly on url => Should redirect to Sign In.
